### PR TITLE
First try to store custom_keras losses etc.

### LINKED ISSUE #17: Storage of custom keras loss function
**Describe the bug**
If pipeline.to_folder is called on a pipeline which contains a KerasWrapper with a custom loss function, then pyWATTS fails. 

**To Reproduce**
1. Build a NN with Keras with a custom loss functoin
2. Add the NN and the custom loss function to the wrapper
3. Add the wrapper to the pipline
4. Call to_folder on the pipeline

**Expected behavior**
The pipeline with the keraswrapper and the custom loss function is stored and can be reloaded correctly.

### PR DESCRIPTION
**Pull request by benHeid**
## Related Issues
closes #17 
## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos)
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Did you update the CHANGELOG

## PR review
Anyone in the community is free to review the PR once the tests have passed.

- [ ] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones to the PR so it can be classified

### CODE REVIEW

**Review by kalebphipps — APPROVED** [review]
Everything looks good and the code ran perfectly locally for me with multiple different custom loss functions.
